python/external_config/external_command.py

```diff
@@ -59,13 +61,44 @@ def serialize_command(cls, entity_type, command_name, properties):
             "group_default": properties.get("group_default") or False,
 
             # special for shotgun
-            "deny_permissions": properties.get("deny_permissions"),
-            "deny_platforms": properties.get("deny_platforms"),
-            "supports_multiple_selection": properties.get("supports_multiple_selection"),
+            "sg_deny_permissions": properties.get("deny_permissions"),
+            "sg_supports_multiple_selection": properties.get("supports_multiple_selection"),
         }
 
         return data
 
+    @classmethod
+    def enabled_on_current_os(cls, properties):
+        """
+        Checks toolkit command properties to determine
+        if a command is enabled on the current os or Not.
+
+        :param dict properties: Properties dictionary
+            as returned by the ``Engine.commands`` dictionary property.
+        :return: True if enabled, False if not.
+        """
+        if "deny_platforms" in properties:
+            # setting can be Linux, Windows or Mac
+            curr_os = {"linux2": "Linux", "darwin": "Mac", "win32": "Windows"}[sys.platform]
```

Comment:
Don't we have a helper for this? I remember having similar code somewhere else, so could be the occasion to have a standard single place to convert `sys.platform` to the names we handle. And this could be cached when this file is imported, rather than being done here.

Reply:
`ShotgunPath` doesn't actually provide any good services for this, i looked into it - and i am not sure it's worth adding.